lms/djangoapps/courseware/tests/test_group_access.py

```diff
@@ -183,8 +184,13 @@ def check_access(self, user, block_location, is_accessible):
         """
         DRY helper.
         """
+        # Set the partition service on the item.
+        item = modulestore().get_item(block_location)
+        item.runtime._services['partitions'] = LmsPartitionService(  # pylint: disable=protected-access
```

Comment:
@cpennington @nasthagiri just hit another partition service unit test problem that I don't know how to solve. It's related to discussion component visibility, and I can't bind a partition service to an actual xblock instance because the unit tests are testing APIs that read all the discussion xblocks from the course module and then iterate through them.